gm4_area_markers/data/area_markers/functions/marker/process.mcfunction

```diff
@@ -0,0 +1,10 @@
+#called from main
+#@s = area marker
+
+#make message apear
+execute at @s[tag=gm4_am_tier_1] positioned ~-24 0 ~-24 run title @a[dx=48, dy=1000000000000, dz=48] actionbar [{"translate":"%1$s%3427655$s","with":["Area: ",{"translate":"text.gm4.area_markers.area"}],"color":"gold"},{"selector":"@s","color":"white"}]

```

Comment:
I definitely feel like a vertical area of 1 billion blocks is definitely far too much here... you're running this from y=0 so 256 (or 300 if you want to hit fools flying above the world) would be perfectly enough.

Reply:
done